rules/skills-management.md

```
@@ -0,0 +1,46 @@
+# Skills Management
+
+External skills are declared in `dotagents/SKILLS.txt` (canonical) and locked
+in `dotagents/skills-lock.json` (generated, committed). Skills install globally
+via the skills CLI (https://github.com/vercel-labs/skills/issues/549).
+
+## SKILLS.txt Format
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)